src/templateModule.tsx

```diff
@@ -4,16 +4,15 @@ import { Form, Input, Grid, Card, Statistic } from 'semantic-ui-react';
 import { useSubstrate } from './substrate-lib';
 import { TxButton } from './substrate-lib/components';
 
-function Main (props) {
+function Main ({ accountPair }: {accountPair: any}) {
   const { api } = useSubstrate();
-  const { accountPair } = props;
 
   // The transaction submission status
   const [status, setStatus] = useState('');
 
   // The currently stored value
-  const [currentValue, setCurrentValue] = useState(0);
-  const [formValue, setFormValue] = useState(0);
+  const [currentValue, setCurrentValue] = useState('0');
+  const [formValue, setFormValue] = useState('0');
```

Comment:
I suspect TypeScript isn't happy about the state being both a `string` and a `number`, hence why you had to change the argument to a `'0'`, correct?

Reply:
That is corrrect